### PR TITLE
DOMA-3208 add access to reset user by user and tests

### PR DESCRIPTION
**Pull request by DanilovMaxim13**

### CODE REVIEW

File: apps/condo/domains/user/schema/ResetUserService.test.js [hunk 1]
```diff
@@ -144,4 +168,4 @@ describe('ResetUserService', () => {
             expect(e.errors[0].name).toContain('AuthenticationError')
```
Comment (SavelevMatthew) [inline apps/condo/domains/user/schema/ResetUserService.test.js:168]: `expectToThrowAuthenticationErrorToResult`

File: apps/condo/domains/user/schema/ResetUserService.test.js [hunk 2]
```diff
@@ -122,14 +122,38 @@ describe('ResetUserService', () => {
...
+    })
+
+    test('user cant reset another user', async () => {
         const client = await makeClientWithNewRegisteredAndLoggedInUser()
+        const client2 = await makeClientWithNewRegisteredAndLoggedInUser()
         const payload = {
             user: { id: client.user.id },
         }
 
         await catchErrorFrom(async () => {
```
Comment (SavelevMatthew) [inline apps/condo/domains/user/schema/ResetUserService.test.js:155]: `expectToThrowAccessDeniedErrorToResult`

File: apps/condo/domains/user/access/ResetUserService.js [hunk 3]
```diff
@@ -3,12 +3,12 @@
  */
 const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
 
-async function canResetUser ({ authentication: { item: user } }) {
+async function canResetUser ({ args: { data: { user: { id } } }, authentication: { item: user } }) {
```
Comment (pahaz) [inline apps/condo/domains/user/access/ResetUserService.js:6]: it's an runtime error! If you don't have a user!

**Review by SavelevMatthew — APPROVED** [review]

File: apps/condo/domains/user/schema/ResetUserService.test.js [hunk 4]
```diff
@@ -122,26 +123,47 @@ describe('ResetUserService', () => {
         })
     })
 
-    test('user cant reset user', async () => {
+    test('user can reset their account', async () => {
```
Comment (toplenboren) [inline apps/condo/domains/user/schema/ResetUserService.test.js:126]: Please add a test that user cant reset other account

**Review by toplenboren — APPROVED** [review]

**Pull request merged**